adaptive_breakpoints/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: adaptive_breakpoints
 description: A Flutter package for adaptive breakpoints following the Material guidelines.
-version: 0.1.1
-homepage: https://github.com/material-components/material-components-flutter-adaptive
+version: 0.1.2
+homepage: https://github.com/material-components/material-components-flutter-adaptive/tree/develop/adaptive_breakpoints
```

Comment:
ah, @guidezpl - I apologize. Here, I'd meant to switch the `homepage` field to the `repository` field.